src/lib/eliom_common.server.mli

```diff
@@ -711,7 +711,7 @@ type eliom_js_page_data = {
 
 val get_site_dir : sitedata -> Url.path
 val get_site_dir_string : sitedata -> string
-val get_secure : bool option -> sitedata -> bool
+val get_secure : secure_o:bool option -> sitedata:sitedata -> unit -> bool
```

Comment:
This is an internal function, and naming it like that helped a lot to check bugs in that code
(because we have optional `secure` sometimes, sometimes `secure` mandatory and bool, sometimes `secure` mandatory and bool option...) 
Anyway this commit is not really important. I did it while tracking a difficult bug and find that it will help in the future (because the code for secure cookies is somewhat tricky).